profiles/arch/riscv/package.use.mask

```diff
@@ -199,3 +199,7 @@ sci-libs/opencascade vtk
 dev-libs/libpcre2 jit
 dev-libs/libpcre jit
 www-servers/nginx pcre-jit
+
+# ArchFeh <guyu2876@gmail.com> (2022-07-26)
```

Comment:
I'd add this entry to the top, as the list is descending by date from top to bottom.

Reply:
Solved. I didn't notice about the order.